frontend/client/src/screens/Login.js

```diff
@@ -30,6 +30,10 @@ const SignInFormBase = observer(
       this.trySignInWithEmailLink()
     }
 
+    componentDidMount() {
```

Comment:
The frontend tests are failing and I believe it because of the interaction between:

1) how the enzyme wrapper is being constructed
2) using `componentDidMount()`

Not totally sure why but I don't believe the frontend test errors are anything to be concerned with?  @colbymorrison thoughts?

Reply:
Have you tried using `React.createRef()`? Works very similar to what you're trying to do, but I think it plays nicer.  
  
Docs: https://reactjs.org/docs/refs-and-the-dom.html

Reply:
I encountered this error before. The issue is actually using `document.getElementById.` The tests use jsdom to imitate the DOM in the browser. However, the imitation is not perfect and `document.getElementById` always returns null in the test environment. See `frontend/client/__tests__/code_validations.test.js` Line 74-80 for how I got around this, but I think Leon is correct, the better approach is to use refs, I think they're the more React-y way to do this.

Reply:
Updated using `createRef` instead.  Thx for the feedback!